hiredis-client/ext/redis_client/hiredis/extconf.rb

```diff
@@ -31,39 +31,32 @@
   end
 
   Dir.chdir(hiredis_dir) do
-    flags = ["static", "USE_SSL=1"]
+    cflags = ["-fvisibility=hidden"]
+    make_args = ["static", "USE_SSL=1"]
     if openssl_lib
-      flags << %(CFLAGS="-I#{openssl_include}") << %(SSL_LDFLAGS="-L#{openssl_lib}")
+      cflags << "-I#{openssl_include}"
+      make_args << %(SSL_LDFLAGS="-L#{openssl_lib}")
     end
 
-    flags << "OPTIMIZATION=-g" if ENV["EXT_PEDANTIC"]
+    make_args << "CFLAGS=#{cflags.join(' ')}"
```

Comment:
Right, there is no need to quote since it's using `system` with multiple args, LGTM